src/Makefile.qt.include

```diff
@@ -360,13 +360,13 @@ translate: $(srcdir)/qt/bitcoinstrings.cpp $(QT_FORMS_UI) $(QT_FORMS_UI) $(BITCO
 $(QT_QRC_LOCALE_CPP): $(QT_QRC_LOCALE) $(QT_QM)
 	@test -f $(RCC)
 	@cp -f $< $(@D)/temp_$(<F)
-	$(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(RCC) -name bitcoin_locale $(@D)/temp_$(<F) | \
+	$(AM_V_GEN) QT_SELECT=$(QT_SELECT) QT_RCC_SOURCE_DATE_OVERRIDE=1 $(RCC) -name bitcoin_locale --format-version 1 $(@D)/temp_$(<F) | \
```

Comment:
If we're using `--format-version 1`, can't we drop `QT_RCC_SOURCE_DATE_OVERRIDE` ?

Reply:
Yes! I've overlooked it :man_facepalming:

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/21654#issuecomment-823347232).